core/lib/zksync_core/src/genesis.rs

```diff
@@ -321,6 +322,29 @@ async fn insert_system_contracts(
 ) -> Result<(), GenesisError> {
     let system_context_init_logs = (H256::default(), get_system_context_init_logs(chain_id));
 
+    let known_code_storage_logs: Vec<_> = contracts
+        .iter()
+        .map(|contract| {
+            let hash = hash_bytecode(&contract.bytecode);
+            let known_code_key = get_known_code_key(&hash);
+            let marked_known_value = H256::from_low_u64_be(1u64);
+            println!(
+                "{:?} : {:?} : {:?}",
+                hex::encode(contract.account_id.address().as_bytes()),
+                hex::encode(hash.as_bytes()),
+                hex::encode(known_code_key.hashed_key().as_bytes())
+            );
```

Comment:
do we need this print?
How it was done before? should we remove it from the old place?

Reply:
it was there for debugging, ill remove it